tests/back/Pim/Enrichment/Specification/Component/Product/Comparator/Filter/EntityWithValuesFilterSpec.php

```diff
@@ -287,12 +282,96 @@ function it_throws_an_exception_if_new_values_are_bad_formatted(
         ];
 
         $normalizer->normalize($product, 'standard')->willReturn($originalValues);
-        $attributeRepository->getAttributeTypeByCodes(array_keys($newValues['values']))->willReturn([
-            'description' => 'pim_catalog_textarea'
-        ]);
+        $getAttributes->forCode(Argument::any())->willReturn($this->getDescriptionAttribute());
 
         $this
             ->shouldThrow(InvalidPropertyTypeException::class)
             ->during('filter', [$product, $newValues]);
     }
+
+    public function it_filters_ignoring_attribute_case(
+        NormalizerInterface $normalizer,
+        ComparatorRegistry $comparatorRegistry,
+        GetAttributes $getAttributes,
+        FilterInterface $productFieldFilter,
+        ProductInterface $product,
+        ComparatorInterface $descriptionComparator
+    ): void {
+        $originalValues = [
+            'family' => 'tshirt',
+            'values' => [
+                'description' => [
```

Comment:
```suggestion
                'DEScription' => [
```

In order to check with mixed case